CODEOWNERS

```diff
@@ -27,11 +28,12 @@ Makefile*                                                        @istio/wg-test-
 /pkg/bootstrap/                                                  @istio/wg-networking-maintainers
 /pkg/config/                                                     @istio/wg-networking-maintainers
 /pkg/envoy/                                                      @istio/wg-networking-maintainers
-/pkg/istio-agent/                                                @istio/wg-networking-maintainers
+/pkg/istio-agent/                                                @istio/wg-networking-maintainers @istio/wg-security-maintainers
 /pkg/keepalive/                                                  @istio/wg-networking-maintainers
 /pkg/kube/                                                       @istio/wg-networking-maintainers
 /pkg/mcp/                                                        @istio/wg-networking-maintainers
 /pkg/queue/                                                      @istio/wg-networking-maintainers
+/pkg/security/                                                   @istio/wg-security-maintainers
```

Comment:
The package is critical for env - since all config and integration is done here. Should have environments-maintainers.

Reply:
This file only contains security configuration, right? It's mostly extracted from security/ folder.

Reply:
Yes, it's only config - so env related ( install, integration, etc ). No actual security logic.

Reply:
But config setting is in pilot/cmd/pilot-agent/main.go. This file is mainly to define the data structure consumed by security package.

Reply:
Costin, I added env maintainer to this file as well. Can you approve it?